packages/2018/src/components/PortlandCollectionPage.js

```diff
@@ -1,16 +1,148 @@
 import React from 'react';
+import { css } from 'emotion';
 import { Link } from 'react-router';
 
+const collectionPageWrapper = css`
+  position: relative;
+  display: block;
+  width: 100%;
+  overflow-y: hidden;
+  h1 {
+    width: 400px;
+    display: inline-block;
+    font-size: 80px;
+    line-height: 1.2;
+    letter-spacing: -1px;
+    font-weight: 400;
+    margin: 0 0 20px;
+    font-family: 'Rubik', sans-serif;
+
+    @media (max-width: 1024px) {
+      font-size: 60px;
+    }
+  }
+`;
+
+const leftContainer = css`
+  position: fixed;
+  width: 60%;
+  padding: 60px;
+  left: 0;
+  box-sizing: border-box;
+
+  @media (max-width: 850px) {
+    position: relative;
+    width: 100%;
+    padding: 24px 36px;
+  }
+`;
+
+const subCopy = css`
+  position: relative;
+  display: block;
+  font-family: 'Rubik',sans-serif;
+  font-size: 18px;
+  line-height: 1.7;
+  margin: 20px 0;
+
+  @media (max-width: 1024px) {
+    font-size: 16px;
+  }
+`;
+
+const sideListWrapper = css`
+  display: block;
+  position: relative;
+  margin-left: 60%;
+
+  @media (max-width: 850px) {
+    margin-left: 0;
+  }
+
+  ul {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    width: 40vw;
+    height: 100vh;
+    background-color: black;
+    min-width: 320px;
+    overflow-y: scroll;
+
+    @media (max-width: 850px) {
+      width: 100%;
+      overflow-y: auto;
+      height: auto;
+    }
+
+    a:nth-child(1) > li {
+      background-color: #DC4556;
+    }
+    a:nth-child(2) > li {
+      background-color: #19B7AA;
+    }
+    a:nth-child(3) > li {
+      background-color: #1E62BD;
+    }
+    a:nth-child(4) > li {
+      background-color: #721D72;
+    }
+    a:nth-child(5) > li {
+      background-color: #FFB226;
+    }
```

Comment:
It's probably better to use class names per project instead of relying on ordering. This way in the event that we want to shuffle the collections, there aren't unforeseen CSS consequences.

From what I understand this is temporary anyway, so maybe you have already gone through this train of that. In that case, disregard!

Reply:
Great point! Yeah it totally makes sense to use class names instead of child ordering. I will update that.